### PR TITLE
add our lessons

### PR DESCRIPTION
**Pull request by VladDlogush**

### CODE REVIEW

File: src/partials/our-lessons.html [hunk 1]
```diff
@@ -0,0 +1,60 @@
+<section class="our-lessons" id="our-lessons">
```
Comment (asaulyak) [inline src/partials/our-lessons.html:1]: add global section class

**Comment by asaulyak** [comment]
image
make content responsive for width < 375

File: src/partials/our-lessons.html (hunk 1 above)
Comment (VladDlogush) [reply src/partials/our-lessons.html]: done

File: src/css/our-lessons.css [hunk 2]
```diff
@@ -0,0 +1,162 @@
...
+  border-radius: 16px;
+  border: 1px solid var(--green);
+  background-color: var(--white);
+  margin-bottom: 40px;
+  padding: 40px 20px;
+  color: var(--soft-black);
+  max-width: 336px;
+}
+
+.our-lessons-item:nth-child(2) {
```
Comment (asaulyak) [inline src/css/our-lessons.css:15]: use nth-child(2n)

File: src/css/our-lessons.css [hunk 3]
```diff
@@ -0,0 +1,162 @@
...
+  font-size: 14px;
+  font-weight: 700;
+  line-height: 1.71;
+  color: var(--white);
+  background-color: var(--green);
+  transition: background-color var(--animation-fast);
+}
+
+.our-lessons-button:hover {
+  background-color: #38ae5f;
```
Comment (asaulyak) [inline src/css/our-lessons.css:71]: use color variable

File: src/css/our-lessons.css [hunk 4]
```diff
@@ -0,0 +1,162 @@
...
+}
+
+.our-lessons-secondary-button {
+  padding: 14px 76px;
+  border: 1px solid rgba(255, 255, 255, 0.4);
```
Comment (asaulyak) [inline src/css/our-lessons.css:76]: check if this color exists in vars
add one if it doesn't

File: src/css/our-lessons.css [hunk 5]
```diff
@@ -0,0 +1,162 @@
...
+  border-radius: 12px;
+  font-size: 14px;
+  font-weight: 700;
+  line-height: 1.71;
+  color: var(--white);
+  background-color: var(--green);
+  transition: background-color var(--animation-fast);
+}
+
+.our-lessons-secondary-button:hover {
```
Comment (asaulyak) [inline src/css/our-lessons.css:87]: add focus

File: src/css/our-lessons.css (hunk 2 above)
Comment (VladDlogush) [reply src/css/our-lessons.css]: done!

File: src/css/our-lessons.css (hunk 3 above)
Comment (VladDlogush) [reply src/css/our-lessons.css]: done!

File: src/css/our-lessons.css (hunk 4 above)
Comment (VladDlogush) [reply src/css/our-lessons.css]: done!

File: src/css/our-lessons.css (hunk 5 above)
Comment (VladDlogush) [reply src/css/our-lessons.css]: done!

File: src/css/our-lessons.css [hunk 6]
```diff
@@ -0,0 +1,164 @@
...
+  margin-bottom: 20px;
+  font-size: 14px;
+  line-height: 1.71;
+  letter-spacing: -0.02em;
+  text-align: center;
+}
+
+.our-lessons-button {
+  padding: 14px 76px;
+  border: 1px solid rgba(255, 255, 255, 0.4);
```
Comment (asaulyak) [inline src/css/our-lessons.css:59]: use color var

File: src/partials/our-lessons.html [hunk 7]
```diff
@@ -0,0 +1,60 @@
...
+      Our <span class="section-title-accent">lessons</span>
+    </h2>
+    <p class="section-description our-lessons-description">
+      Our lessons are carefully structured and interactive, designed to promote
+      rapid language acquisition through practical tasks and real-world
+      scenarios
+    </p>
+    <ul class="our-lessons-list">
+      <li class="our-lessons-item">
+        <h3 class="our-lessons-item-title our-lessons-title-a black-color">
```
Comment (asaulyak) [inline src/partials/our-lessons.html:13]: replace h3 with p

File: src/css/our-lessons.css (hunk 6 above)
Comment (VladDlogush) [reply src/css/our-lessons.css]: done!

File: src/partials/our-lessons.html (hunk 7 above)
Comment (VladDlogush) [reply src/partials/our-lessons.html]: done!

**Review by asaulyak — APPROVED** [review]

**Pull request merged**